src/sharedHelpers/geolocationWrapper.e2e-mock

```diff
@@ -1,31 +1,25 @@
-import Geolocation, {
-  GeolocationError,
+import {
   GeolocationResponse
 } from "@react-native-community/geolocation";
 import { CHUCKS_PAD } from "appConstants/e2e.ts";
 
+let counter = 0;
+
 function watchPosition(
-  success: ( position: GeolocationResponse ) => void,
-  error?: ( error: GeolocationError ) => void,
-  options?: {
-    interval?: number;
-    fastestInterval?: number;
-    timeout?: number;
-    maximumAge?: number;
-    enableHighAccuracy?: boolean;
-    distanceFilter?: number;
-    useSignificantChanges?: boolean;
-  }
+  success: ( position: GeolocationResponse ) => void
 ) {
   console.log( "[DEBUG geolocationWrapper.e2e-mock] watchPosition" );
   const watchID = Date.now();
-  setTimeout( ( ) => {
-    console.log( "[DEBUG geolocationWrapper.e2e-mock] watchPosition success" );
-    success( {
-      coords: CHUCKS_PAD,
-      timestamp: Date.now()
-    } );
-  }, 1000 );
+  if ( counter < 5 ) {
```

Comment:
We have to limit this here to run only five times otherwise the e2e test never idles and times out.